### PR TITLE
DevTools 107 > Memory Inspector > Wasm object memory

### PR DESCRIPTION
**Pull request by sofiayem**
Documented https://developer.chrome.com/en/blog/new-in-devtools-107/#memory

Publish on Oct 25.

### CODE REVIEW

**Comment by jecfish** [comment]
@ktran can you help to review this documentation?

**Review by ktran — APPROVED** [review]
LGTM, thanks a lot!

File: site/en/docs/devtools/memory-inspector/index.md [hunk 1]
```diff
@@ -1,23 +1,24 @@
...
-title: "Memory Inspector: Inspect JavaScript ArrayBuffer"
+title: "Memory Inspector: Inspect ArrayBuffer, TypedArray, DataView, and Wasm Memory."
 authors:
   - jecelynyeen
+  - sofiayem
 date: 2020-08-20
-#updated: YYYY-MM-DD
-description: "Use the Memory inspector to inspect an ArrayBuffer in JavaScript, as well as a WebAssembly.Memory"
+updated: 2022-10-25
+description: "Use the Memory inspector to inspect an ArrayBuffer, TypedArray, DataView in JavaScript as well as WebAssembly.Memory of C++ Wasm apps."
```
Comment (rachelandrew) [inline site/en/docs/devtools/memory-inspector/index.md:9]:
```suggestion
description: "Use the Memory inspector to inspect an ArrayBuffer, TypedArray, or DataView in JavaScript as well as WebAssembly.Memory of C++ Wasm apps."
```

File: site/en/docs/devtools/memory-inspector/index.md [hunk 2]
```diff
@@ -102,16 +103,40 @@ Let's inspect the memory together.
     {% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/RqOAPGuSPUReZ6yRIf2Z.png", alt="change the encoding", width="800", height="602" %}
```
Comment (rachelandrew) [inline site/en/docs/devtools/memory-inspector/index.md:103]:
```suggestion
    {% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/RqOAPGuSPUReZ6yRIf2Z.png", alt="Change the encoding.", width="800", height="602" %}
```

**Review by rachelandrew — APPROVED** [review]

**Pull request merged**